clang/include/clang/Format/Format.h

```diff
@@ -2432,9 +2431,25 @@ struct FormatStyle {
   ///    label2:                                label2:
   ///      return 1;                              return 1;
   ///    }                                      }
+  ///
+  ///    GLI_HalfIndent:
```

Comment:
where in hte code is GLI_HalfIndent handled differently from GLI_Indent sorry I'm confused

Reply:
https://github.com/llvm/llvm-project/issues/24492#issuecomment-980948078

https://github.com/llvm/llvm-project/issues/24492#issuecomment-980948080

I added this based on the above two